src/proto/streams/prioritize.rs

```diff
@@ -530,6 +530,7 @@ impl Prioritize {
                     trace!("pop_frame; stream={:?}", stream.id);
 
                     let is_counted = stream.is_counted();
+                    let is_pending_reset = stream.is_pending_reset_expiration();
```

Comment:
This line was confusing at first. My initial thought was that, at this point, the stream could never be pending reset expiration. However, a stream transitions to `pending_reset_expiration` immediately when the client resets the stream. At this point, the `RST_STREAM` frame is still queued, and this is probably the point at which we are reading it.

So, in fact, it is actually expected for `is_pending_reset` to be true sometimes.

Could you add some comments describing this?